backstop/tests/stepper.html

```diff
@@ -0,0 +1,199 @@
+<!--
```

Comment:
Have you tried using `git mv` command so that it wouldn't show this as a new file?

Reply:
Made a new PR: https://github.com/iTwin/iTwinUI/pull/642